package/yast2-update.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-update
-Version:        4.0.17
+Version:        4.0.18
```

Comment:
The fix is a bit different than in the SLE15 branch and diverges, please bump the version to `4.1.0`.